lib/db.js

```diff
@@ -257,6 +257,26 @@ module.exports = function (
       )
   }
 
+  DB.prototype.sessions = function (uid) {
```

Comment:
If it's not too much a PITA, can you please do the addition of this `sessions` function as a separate commit?  It will make the auto-generated changelog a bit more comprehensive.